druid/src/widget/list.rs

```diff
@@ -45,6 +165,41 @@ pub struct List<T> {
 impl<T: Data> List<T> {
     /// Create a new list widget. Closure will be called every time when a new child
     /// needs to be constructed.
+    ///
+    /// The list widget can have a function passed in as a closure, or an abstract closure can also
+    /// be provided.
+    ///
```

Comment:
This part seems superfluous. I think you can assume the vast majority of Rust users will be familiar with how closures work. (And if they aren't, this isn't the place to explain them)